From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: 1vivy <1vivy@tutanota.com>
Date: Sat, 11 Jul 2026 16:55:00 -0600
Subject: [PATCH] libhwy: expose the static library to vendor modules

libjxl is vendor-available and links libhwy statically. Expose libhwy to the
vendor image so Soong can create the required vendor/static variant.
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
--- a/Android.bp
+++ b/Android.bp
@@ -18,6 +18,7 @@ license {
 cc_library {
     name: "libhwy",
     host_supported: true,
+    vendor_available: true,
     sdk_version: "current",
     stl: "c++_shared",
     export_include_dirs: [
--
2.49.0
